TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -114,6 +114,21 @@ public void channelUnregistered(ChannelHandlerContext ctx) throws Exception {
         });
     }
 
+    @Override
+    public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
```

Comment:
Why do we need this function now?

Reply:
To close the serializer - see the previous comment that you left :)